rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml

```diff
@@ -27,7 +27,8 @@ fields:
     - CommandLine
     - ParentCommandLine
 falsepositives:
-    - Unknown
+    - During Anaconda update the 'conda.exe' process will eventually launch the command described in the detection section
```

Comment:
Can you describe the FP as a filter selection?

Reply:
I'll add a filter but I would have to rely on the grandparent field which is not available in sysmon EID 1 and 4688. (I'll simply add a note that this won't work without aurora, similar to how did in other rules)

Reply:
ok that's suboptimal but if we cannot filter differently, it is as it is.